Functions

```diff
@@ -0,0 +1,24 @@
+Functions
```

Comment:
You can use `//` to add comments that don't get interpreted by the runtime environment. For example:

```
// Functions
```